resources/views/contact-us.blade.php

```diff
@@ -0,0 +1,119 @@
+<x-app-layout>
+
+        <h2 class="text-lg font-formula1 py-3">Contact Us</h2>
+
+    <div class="grid grid-cols-1 md:grid-cols-2 grid-rows-1 gap-4 ">
```

Comment:
Add padding to the grid. See how you can add padding here: [Tailwind CSS: Padding](https://tailwindcss.com/docs/padding).